source/blog/2018-06-08-the-emberjs-times-issue-50.md

```diff
@@ -13,9 +13,14 @@ Read either on the [Ember blog](https://www.emberjs.com/blog/2018/06/08/the-embe
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Lets Send SendAction Away!](#lets-send-send-action-away)
```

Comment:
Currently we use the header titles to link to the respective url that we find is most important in a particular section. Therefore I think this would be

```md
## [Lets Send SendAction Away!](https://github.com/emberjs/rfcs/pull/335)
```

What do you think?

Reply:
Looks good. I went be near a computer until tomorrow as it's pretty late for me. Don't think I'll have time to rebase. I'd love to have some help :)